pkg/common/fflag/fflag_test.go

```diff
@@ -0,0 +1,157 @@
+package fflag
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/hashicorp/hcl"
+	"github.com/hashicorp/hcl/hcl/ast"
+)
+
+const (
+	flagTestFlag Flag = "i_am_a_test_flag"
```

Comment:
super happy this is now scoped to just the test!!!

Reply:
I'm not sure it can stay here 😭 I think I need to put it back .. still working through it ... we want to be able to assert that the statically defined common flags get loaded up, and to do that I think we need one defined there that we can test against. Since it is defined at the package level, I've had a hard time figuring out how to inject/manipulate it cleanly from the test suite

Reply:
> Since it is defined at the package level, I've had a hard time figuring out how to inject/manipulate it cleanly from the test suite

I'm specifically talking about the commonFlagMap var, not the flag itself

Reply:
Ah, for that, you can leverage an `init()` function in the `fflag_test.go` file to mutate the commonFlagMap just for tests without mutating it for production use.

```
func init() {
    commonFlagMap[flagTestFlag] = true
}
```

Reply:
oh neat! That's a good idea, I will try it out